### ISSUE TITLE
Negative margins

### ISSUE THREAD

**Issue by baranwang** [romashamin/compo-sketch#9]
I need padding less than 0, like "l:-4", the element is center of BG

**Comment by illustree** [comment]
yes please! would be appreciated ... need that also often and then always tweak manually
